static/src/javascripts-legacy/projects/common/views/svgs.js

```diff
@@ -145,10 +143,11 @@ define([
     };
 
     return function (name, classes, title) {
-        if (config.switches.webpack) {
+        // #wp-rjs - needed by karma
+        if (svgs[name].markup) { // webpack way
```

Comment:
I second that! I started a [weekly cleanup](https://github.com/guardian/frontend/pull/15841) and can take care of the commercial ones next Wednesday 😃  Also we are phasing out all the commercial templates so they will eventually all go away.